larky/src/main/resources/stdlib/xml/etree/ElementTree.star

```diff
@@ -931,8 +939,14 @@ def _namespaces(elem, default_namespace=None):
                 .build()
 
     # populate qname and namespaces table
-    for elem in elem.iter():
-        tag = elem.tag
+    # for elem in elem.iter():
+    qu = [elem]
```

Comment:
Great job going BFS instead of recursion. This is a great pattern. Can you document it in https://github.com/mahmoudimus/py2star/blob/main/MIGRATING.md?

Reply:
will do